### DISCUSSION TITLE
Request Timeout

### DISCUSSION THREAD

**Discussion by am-abudu** [community/community#19545] [GitHub Education] [answered]
![image](https://user-images.githubusercontent.com/51912589/175928120-ecf969e6-c415-44f1-8861-527e4a9d442d.png)
It said that I should receive an email from us by Jun 26. But it's Jun 27 today and I didn't receive an email.
My email is ******.

**Comment by melvincwng** [comment, marked as answer]
@am-abudu, can you kindly raise a ticket to [GitHub Education Support](https://support.github.com/contact/education) for them to investigate this issue?

Hope this clarifies.

Thank you, and have a nice day ahead 😄

**Reply by melvincwng** [reply]
@am-abudu, thanks for marking this as answer.

My suggestion would be to wait patiently for a couple of days more for the support engineers to check on your issue.

Feel free to follow me on GitHub if you want :)

Hope you get your approval as soon as possible for the Github Education Student Developer Pack 😄

**Comment by samus-aran** [comment]
Hi @am-abudu ,

I've removed your private information from [this post](https://github.com/community/community/discussions/19545) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
